docker-compose-setup/init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.." || exit
+mvn package -DskipTests
+docker run --rm -dit -v coatrack-admin:/home --name coatrack-admin ubuntu
+docker cp ./spring-boot/proxy/target/coatrack-proxy-2.0.0-SNAPSHOT.jar coatrack-admin:/home
```

Comment:
This would probably become obsolete after we add the jar to the docker image, right?

Reply:
Yes. This part is already removed in #38.